recipe/meta.yaml

```diff
@@ -18,15 +18,15 @@ build:
     - pip install --no-deps tensorboard=={{ version }}
 
 requirements:
-  # See: https://pypi.python.org/pypi/tensorflow-tensorboard/0.4.0rc3
+  # See: https://pypi.org/project/tensorboard
   build:
     - python
     - pip
   run:
     - python
     - futures >=3.1.1  # [py27]
-    - bleach ==1.5.0
-    - html5lib ==0.9999999
+    - bleach >=1.5.0
+    - html5lib >=0.9999999
     - markdown >=2.6.8
     - numpy >=1.12.0
     - protobuf >=3.4.0
```

Comment:
Is there a particular reason to deviate from the versions specified in the `setup.py`?
https://github.com/tensorflow/tensorboard/blob/1.8.0/tensorboard/pip_package/setup.py#L25-L40
```python
REQUIRED_PACKAGES = [
    'numpy >= 1.12.0',
    'six >= 1.10.0',
    'protobuf >= 3.4.0',
    'werkzeug >= 0.11.10',
    'html5lib == 0.9999999',  # identical to 1.0b8
    'markdown >= 2.6.8',
    'bleach == 1.5.0',

    # futures is a backport of the python 3.2+ concurrent.futures module
    'futures >= 3.1.1; python_version < "3"',

    # python3 specifically requires wheel 0.26
    'wheel; python_version < "3"',
    'wheel >= 0.26; python_version >= "3"',
]
```
Also, it looks like `six` and `werkzeug` need to be added.

Reply:
bleach in particular introduced incompatible changes in the 2.0 release which are not expected to work with tensorboard, tensorflow/tensorboard#427.  The good news is that html5lib and bleach are vendored into tensorboard for the next release, tensorflow/tensorboard#1142.